### ISSUE TITLE
'computed' property always false in async object expression methods

### ISSUE THREAD

**Issue by axefrog** [cherow/cherow#264]
**version: 1.6.8**

Test case:

```js
const key = 'bar';
const foo = { async [key] () { } };
```

Output:

```js
output = {
  type: 'Property',
  key: { type: 'Identifier', name: 'key' },
  value:
    { type: 'FunctionExpression',
      params: [],
      body: { type: 'BlockStatement', body: [] },
      async: true,
      generator: false,
      expression: false,
      id: null },
  kind: 'init',
  computed: false,
  method: true,
  shorthand: false }
```

As you can see, `computed` is false, but should be true. I've tried with generator methods, but the issue only seems to occur when the `async` keyword is present.

**Update:** Just noticed this one also produces an incorrect value for `computed`:

```js
const foo = {
  get ['bar'] () { },
};
```

Perhaps it's something to do with property keys that are preceded by a keyword?

**Comment by KFlash** [comment]
Added a fix for this to fix it in 1.6.x. It's non existing in upcoming 2.0.

**Comment by axefrog** [comment]
Thanks for fixing that!

**Comment by KFlash** [comment]
Closing as it will be fixed in the upcoming beta release of 2.0
